### DISCUSSION TITLE
[Plugin] [MongoDB]

### DISCUSSION THREAD

**Discussion by shenlq1017** [devlive-community/datacap#296] [General]
现在的源代码中，mongodb使用的是jdbc的写法，请问这本身能测试通过？是不是代码没完善，demo网站上也无法测试联通。

**Comment by qianmoQ** [comment]
源码中的mongodb已经经过docker测试。如果使用中出现问题请提供详细信息，demo网站已经扫描到数据库被测试的用户删除，正在处理解决
